cake/BuildExternals.cake

```diff
@@ -699,13 +699,17 @@ Task ("externals-download")
     if (string.IsNullOrEmpty (AZURE_BUILD_ID))
         throw new Exception ("Specify a build ID with --azureBuildId=<ID>");
 
-    var url = string.Format(AZURE_BUILD_URL, AZURE_BUILD_ID, "native");
+    var artifactName = "native-default";
```

Comment:
Using variables - wise move.

Reply:
Thanks again for your help and for maintaining this project!